application/controllers/users.py

```diff
@@ -0,0 +1,73 @@
+from flask_restful import Resource, reqparse, request
+from flask_restful import fields, marshal_with, marshal
+from application.models.user import User
+from application import db
+
+user_fields = {
+    'id': fields.Integer,
```

Comment:
Based on Google Oauth documentation, I think we don't want to store the user id in the back-end, but just the token instead (See https://developers.google.com/identity/sign-in/web/sign-in#get_profile_information note at bottom of section)